src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -99,43 +99,53 @@ def llm_classify(
     prompt_options = PromptOptions(provide_explanation=provide_explanation)
     prompts = map_template(dataframe, eval_template, options=prompt_options)
 
-    labels: List[str] = []
-    explanations: List[Optional[str]] = []
+    labels: List[Union[str, None]] = [None] * len(dataframe)
+    explanations: List[Union[str, None]] = [None] * len(dataframe)
 
     printif(verbose, f"Using prompt:\n\n{eval_template.prompt(prompt_options)}")
     if generation_info := model.verbose_generation_info():
         printif(verbose, generation_info)
 
-    for prompt in tqdm(prompts):
-        with set_verbosity(model, verbose) as verbose_model:
-            response = verbose_model(prompt, instruction=system_instruction, **model_kwargs)
-        if not use_openai_function_call:
-            if provide_explanation:
-                unrailed_label, explanation = (
-                    eval_template.extract_label_from_explanation(response),
-                    response,
-                )
-                printif(
-                    verbose and unrailed_label == NOT_PARSABLE,
-                    f"- Could not parse {repr(response)}",
-                )
+    # Wrap the loop in a try / catch so that we can still return a dataframe
+    # even if the process is interrupted
+    try:
+        for index, prompt in tqdm(enumerate(prompts)):
```

Comment:
otherwise it looks like this (with `?` as denominator)
<img width="178" alt="Screenshot 2023-11-15 at 9 36 03 PM" src="https://github.com/Arize-ai/phoenix/assets/80478925/33b9df57-4b39-4984-aa4b-984b1db81f8c">

```suggestion
        for index, prompt in enumerate(tqdm(prompts)):
```